scenario_configs/xview_frcnn_robust_dpatch_defended.json

```diff
@@ -0,0 +1,76 @@
+{
+    "_description": "XView object detection, contributed by MITRE Corporation",
+    "adhoc": null,
+    "attack": {
+        "knowledge": "white",
+        "kwargs": {
+            "batch_size": 1,
+            "brightness_range": [
+                0.8,
+                1.0
+            ],
+            "learning_rate": 0.01,
```

Comment:
@yusong-tan I think there's a minor discrepancy here between the `"learning_rate"` and `"max_iter"` compared to the T&E Eval 3 doc. The latter has:

```
"learning_rate": 0.02,
"max_iter": 100,
```

Is this intentional?

Reply:
@lcadalzo Yes. In the document, I updated the parameters to match those that seem to work well during the baseline evaluations.